src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsFloorAdhesionSkinContext.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers;
+
+public class AdsFloorAdhesionSkinContext {
+    private final String OASIS_CREATIVE_ID = "66223683972";
```

Comment:
I would change it from `private final` to `private static` however it came from old code.